jobs/ci-kubernetes-e2e-gce-gci-latest-1.5-upgrade-etcd.env

```diff
@@ -0,0 +1,15 @@
+### job-env
+
+STORAGE_BACKEND=etcd2
+ETCD_VERSION=2.2.1
+ETCD_IMAGE=2.2.1
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:EtcdUpgrade\] --etcd-upgrade-storage=etcd3 --etcd-upgrade-version=3.0.14
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
```

Comment:
@krzyzacy - what is this one?

IIUC, JENKINS_PUBLISHED_VERSION is the version from which we start upgrade test, right?
But what is JENKINS_PUBLISHED_SKEW_VERSION ?